### DISCUSSION TITLE
Include lib

### DISCUSSION THREAD

**Discussion by fdominik98** [cutelyst/simple-mail#62] [Show and tell]
Hi! Can you write a short tutorial how to include this library to a Qt project on linux?
Thank you in advance!

**Comment by dantti** [comment]
https://github.com/cutelyst/cutelyst/blob/master/Cutelyst/Plugins/View/CMakeLists.txt#L33
then link
https://github.com/cutelyst/cutelyst/blob/master/Cutelyst/Plugins/View/Email/CMakeLists.txt

**Reply by Nelson-Waffo** [reply]
what is someone is using Qt Creator? 

I am having an error: "undefined reference to __imp__ZN10..."
